### PR TITLE
task 25 is completed

### PR DESCRIPTION
**Pull request by scredocoder009**

### CODE REVIEW

**Review by Muhammadislom99 — APPROVED** [review]

File: src/Domain/Entities/TestResult.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+namespace MRA.Jobs.Domain.Entities
+{
+    public class TestResult : BaseAuditableEntity
+    {
+        public long ApplicationId { get; set; }
+        public DateTime CompletedAt { get; set; }
+        public long Id { get; set; }
+        public bool Passed { get; set; }
+        public long Score { get; set; }
```
Comment (IslomMakhsudov) [inline src/Domain/Entities/TestResult.cs:15]: the type of Score doesn't have to be long i think.

File: src/Domain/Entities/Test.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+namespace MRA.Jobs.Domain.Entities;
+public class Test : BaseAuditableEntity
+{
+    public string Description { get; set; }
+    public string Duration { get; set; }
+    public long NumberOfQuestion { get; set; }
+    public string Title { get; set; }
+    public DateTime UpdatedAt { get; set; }
+    public long VacancyId { get; set; }
+    public long PassingScore { get; set; }
```
Comment (IslomMakhsudov) [inline src/Domain/Entities/Test.cs:16]: the type of PassingScore should be int

**Review by IslomMakhsudov — COMMENTED** [review]
Fix your code

**Review by IslomMakhsudov — APPROVED** [review]
Good job!

**Review by Muhammadislom99 — APPROVED** [review]

**Comment by scredocoder009** [comment]
jjj

**Comment by Muhammadislom99** [comment]
good job

**Pull request merged**